pkg/infra/remotecache/redis_storage.go

```diff
@@ -42,17 +42,11 @@ func (s *redisStorage) Get(key string) (interface{}, error) {
 
 	if err == nil {
 		return item.Val, nil
-	}
-
-	if err.Error() == "EOF" {
+	} else if err.Error() == "EOF" {
```

Comment:
Is this required? I dont think if/if else/else statements read better then normal if statements when all if statements return

Reply:
This should be fixed now. PTAL.